src/commands/validators_exit.py

```diff
@@ -115,24 +140,31 @@ def validators_exit(
 async def main(count: int | None) -> None:
     keystores = load_keystores()
     remote_signer_config = None
-
     fork = await consensus_client.get_consensus_fork()
 
     if len(keystores) > 0:
         all_validator_pubkeys = list(keystores.keys())  # pylint: disable=no-member
     else:
-        if not settings.remote_signer_url:
-            raise RuntimeError('No keystores and no remote signer URL provided')
+        if not settings.remote_signer_url and not settings.hashi_vault_url:
```

Comment:
same as before

Reply:
Done in https://github.com/stakewise/v3-operator/pull/184/commits/c8ac6ce3ec65cc6a24c66301a0e34f4277447e2c, thanks